crates/shared/src/price_estimation/buffered.rs

```diff
@@ -35,41 +33,36 @@ pub struct Configuration {
     pub max_batch_len: usize,
     /// An additional minimum delay to wait for collecting requests.
     ///
-    /// The delay to start counting after receiving the first request.
-    pub debouncing_time: Duration,
+    /// The delay starts counting after receiving the first request.
+    pub batch_delay: Duration,
```

Comment:
Are these changes intended or missing a rebase?

Reply:
Not intended. Must have messed up when rebasing. :/